contracts/peg/VanillaPriceBoundPSM.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {PegStabilityModule, Decimal, SafeERC20, SafeCast, IERC20, IPCVDeposit, Constants} from "./PegStabilityModule.sol";
+import {IPriceBound} from "./IPriceBound.sol";
+import {VanillaPSM} from "./VanillaPSM.sol";
+
+/// @notice contract to create a price bound DAI PSM
+/// This contract will allow swaps when the price of DAI is between 98 cents and 1.02 by default
+/// These defaults are changeable by the admin and governance by calling floor and ceiling setters
+/// setOracleFloor and setOracleCeiling
+contract VanillaPriceBoundPSM is VanillaPSM, IPriceBound {
+    using Decimal for Decimal.D256;
+    using SafeERC20 for IERC20;
+    using SafeCast for *;
+
+    /// @notice the default minimum acceptable oracle price floor is 98 cents
+    uint256 public override floor;
+
+    /// @notice the default maximum acceptable oracle price ceiling is $1.02
+    uint256 public override ceiling;
```

Comment:
change to `uint128`

Reply:
floor and ceiling here are following the `IPriceBoundPSM` interface, where it expects `floor()` and `ceiling` both to return `uint256`, we could create a new interface for this to follow though, as changing it will be a breaking change on the current `PriceBoundPSM`

Reply:
that is fine to create a new interface